### PR TITLE
9.0 purchase analytic

### PR DESCRIPTION
**Pull request by JonathanNEMRY**
back port of https://github.com/OCA/account-analytic/pull/97

### CODE REVIEW

**Comment by JonathanNEMRY** [comment]
cc @Cedric-Pigeon

File: purchase_analytic/README.rst [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+Usage
+=====
+
+To use this module, you need to:
+
+* Go to ...
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/142/8.0
```
Comment (Cedric-Pigeon) [inline purchase_analytic/README.rst]: should be 9.0

File: purchase_analytic/README.rst [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+===========
+
+Bugs are tracked on `GitHub Issues
+<https://github.com/OCA/purchase-workflow/issues>`_. In case of trouble, please
+check there if your issue has already been reported. If you spotted it first,
+help us smashing it by providing a detailed and welcomed `feedback
+<https://github.com/OCA/
+purchase-workflow/issues/new?body=module:%20
+purchase_analytic%0Aversion:%20
+8.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
```
Comment (Cedric-Pigeon) [inline purchase_analytic/README.rst]: should be 9.0 and no line break

**Review by Cedric-Pigeon — APPROVED** [review]

File: purchase_analytic/README.rst [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+=================
+
+The goal of this module is to ease analytic account management on purchase order.
+This module add analytic account on purchase order.
+
+If all lines of the purchase order have the same analytic account, the analytic account on the purchase order is automatically set with this value.
+If a analytic account is set on the purchase order, all lines of the purchase will take this value.
+
+Installation
+============
```
Comment (AaronHForgeFlow) [inline purchase_analytic/README.rst:16]: I'd remove empty section

File: purchase_analytic/models/purchase.py [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+
+class PurchaseOrder(models.Model):
+    _inherit = 'purchase.order'
+
+    project_id2 = fields.Many2one(
+        comodel_name='account.analytic.account',
+        string='Contract / Analytic',
+        help='Use to store the value of project_id if there is no lines')
+    project_id = fields.Many2one(
```
Comment (AaronHForgeFlow) [inline purchase_analytic/models/purchase.py:15]: Not sure if this is a good name. It refers to the analytic account not the project.

**Review by AaronHForgeFlow — COMMENTED** [review]
Functional test ok. Just some comments
Edit: As this module is merged in 10 i don't think it's good to change fields name here. So I dismiss this review.

**Review by AaronHForgeFlow — APPROVED** [review]
:+1: Working fine

File: purchase_analytic/README.rst [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+To install this module, you need to:
+
+* Click on install button
+
+Usage
+=====
+
+To use this module, you need to:
+
+* Go to ...
```
Comment (LoisRForgeFlow) [inline purchase_analytic/README.rst:27]: Could you fill the template?

File: purchase_analytic/README.rst [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/142/9.0
+
+Known issues / Roadmap
+======================
+
+* ...
```
Comment (LoisRForgeFlow) [inline purchase_analytic/README.rst:36]: If none, remove this.

File: purchase_analytic/models/purchase.py [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+# © 2016  Laetitia Gangloff, Acsone SA/NV (http://www.acsone.eu)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import api, fields, models
+
+
+class PurchaseOrder(models.Model):
+    _inherit = 'purchase.order'
+
+    project_id2 = fields.Many2one(
```
Comment (LoisRForgeFlow) [inline purchase_analytic/models/purchase.py:11]: I think that the same behavior can be obtained without this workaround...

**Review by LoisRForgeFlow — CHANGES_REQUESTED** [review]
Functionally works ok, just some comments on code.

**Comment by feketemihai** [comment]
@JonathanNEMRY I think first we should have the module merged in 8, and after to migrate it, check also my comment on #55 .

**Comment by AaronHForgeFlow** [comment]
@JonathanNEMRY can you attend the minor comments here? Thanks

**Pull request closed without merging**